devito/ir/iet/nodes.py

```diff
@@ -1077,7 +1077,8 @@ def expr_symbols(self):
             ret.extend(flatten(i.free_symbols for i in self.pointee.symbolic_shape[1:]))
             ret.extend(self.pointer.free_symbols)
         else:
-            assert issubclass(self.pointer._C_ctype, ctypes._Pointer)
+            assert isinstance(self.pointer, LocalCompositeObject) or \
```

Comment:
nitpicking, for code homogeneity : for multi-line statements, instead of the backslash, we use parenthesis for perfectly aligned indentation

```
assert (isinstance(....) or
             issubclass(...))
```